lrathoni/Lullaby-for-a-fish
# [Fix] #12 : Note are now cancelled when overlapping.

This thing here should solve issue #12. My bad.

### What:

* Refactored Command pooling in FlutePlayer.
* Allowed Commands to cancel each other.
* Added an event on Command cancellation.

### To test:

* Checkout this branch.
* Launch play mode in Hub scene.
* Check that the notes are properly cancelled when releasing a button quickly after pressing it.
